docker-compose.yml

```diff
@@ -4,9 +4,16 @@ services:
   web:
     stdin_open: true
     tty: true
-    image: "mmmikem/car-wash-app:0.0.5"
+    build:
+      context: .
+      target: dev
     ports:
-      - "3000:3000"
+      - "3001:3000"
+    command: "./bin/start_dev_server"
+    volumes:
+      - ".:/usr/src/app"
+      - bundle:/bundle
+      - /usr/src/app/tmp
```

Comment:
There's a tmpfile mounting you can do with docker-compose. Don't know if that's something you maybe want to use here?